src/js/node/net.js

```diff
@@ -537,6 +552,7 @@ const Socket = (function (InternalSocket) {
     _final(callback) {
       this[bunSocketInternal]?.end();
       callback();
+      process.nextTick(closeNT, this);
```

Comment:
Is there a scenario where `"close"` can be emitted twice due to the use of `nextTick`?